src/item.cpp

```diff
@@ -5990,5 +5990,5 @@ bool item_category::operator!=( const item_category &rhs ) const
 }
 
 bool item::is_filthy() const {
-    return has_flag( "FILTHY" );
+    return has_flag( "FILTHY" ) && !ACTIVE_WORLD_OPTIONS[ "NO_FILTHY" ];
```

Comment:
There's an inconsistency here. When the mod is applied, you can still wash filthy clothes (only the flag is checked), but cannot see if they're filthy or not in the description (because `!ACTIVE_WORLD_OPTIONS[ "NO_FILTHY" ]` prohibits it here).

The mod description says `"Zombies will nolonger drop filthy clothing"`. To fulfil the description you can check `!ACTIVE_WORLD_OPTIONS[ "NO_FILTHY" ];` before adding the flag, but not here.